docs/howto.rst

```diff
@@ -126,101 +125,146 @@ This is outputted to the terminal:
     INFO:phys2bids.physio_obj:Checking number of timepoints
-    INFO:phys2bids.physio_obj:Found just the right amount of timepoints!
+    WARNING:phys2bids.physio_obj:Found 1 timepoints less than expected!
+    WARNING:phys2bids.physio_obj:Correcting time offset, assuming missing timepoints are at the beginning (try again with a more conservative thr)
     INFO:phys2bids.phys2bids:Plot trigger
     INFO:phys2bids.phys2bids:Preparing 1 output files.
     INFO:phys2bids.phys2bids:Exporting files for freq 1000.0
     INFO:phys2bids.phys2bids:
     ------------------------------------------------
-    Filename:            tutorial_file.txt
+    Filename:            tutorial_file_v2.txt
 
     Timepoints expected: 158
-    Timepoints found:    158
+    Timepoints found:    157
     Sampling Frequency:  1000.0 Hz
-    Sampling started at: 0.24499999999989086 s
+    Sampling started at: 0.2459999999998672 s
     Tip: Time 0 is the time of first trigger
     ------------------------------------------------
 
-.. image:: _static/tutorial_file_trigger_time.png
-   :alt: tutorial_file_trigger_time
+There is one trigger that ``phys2bids`` couldn't find automatically. We can work out that this is the first trigger, if we look at the png file that has been outputted:
+
+.. image:: _static/tutorial_file_v2_trigger_time_Tmissing.png
+   :alt: tutorial_file_channels
    :align: center
-    
 
-Alright! Now we have some outputs that make sense.
-The main difference from the previous call is in **tutorial_file.log** and **tutorial_file_trigger_time.png**.
-The first one now reports 158 timepoints expected (as input) and found (as correctly estimated) and it also tells us that the sampling of the neuroimaging files started around 0.25 seconds later than the physiological sampling.
-The second file contais two plots:
-In the first row, there's the whole trigger channel in blue, an orange block that shows were the time starts (if not there could be something wrong with the threshold). 
-In the second row, we see the first and last trigger (or expected first and last).
+By looking at this figure, we can work out that we need a smaller threshold in order to include the first time point. This can be implemented with the ``-thr`` argument:
 
-**Note**: It is *very* important to calibrate the threshold in a couple of files. This still *won't* necessarily mean that it's the right threshold for all the files, but there's a chance that it's ok(ish) for most of them.
+..  code-block:: shell
 
+    phys2bids -in tutorial_file_v2.txt -indir /home/arthurdent/git/phys2bids/phys2bids/tests/data/ -chtrig 1 -ntp 158 -tr 1.2 -thr 1.04 -outdir /home/arthurdent/physio_v2
+
+    INFO:phys2bids.physio_obj:Counting trigger points
+    INFO:phys2bids.physio_obj:The number of timepoints according to the std_thr method is 158. The computed threshold is 1.04
```

Comment:
@vinferrer: This is not about the docs, but let's change it in this PR: if the manual threshold is inputted, this message should say 
`The number of timepoints found with the manual threshold of 1.04 is 158.`